src/main/resources/assets/minecolonies/gui/windowhutguardtower.xml

```diff
@@ -32,7 +32,7 @@
                          source="minecolonies:textures/gui/builderhut/builder_button_medium.png"
                          label="$(com.minecolonies.coremod.gui.workerhuts.retrieveOff)"/>
 
-            <label size="164 11" pos="13 138" color="black" textalign="MIDDLE"
```

Comment:
what? the pos_x under 13 make no sense as I remember

Reply:
Why not?

Reply:
since the readable area of the builder_paper is 13|size-13 in x dimension

Reply:
and again it's missaligned :D 10px change causes -5px in pos

Reply:
meh, too late :p